common-travel.lic

```diff
@@ -75,10 +75,7 @@ module DRCT
   def retreat(ignored_npcs = [])
     return if (DRRoom.npcs - ignored_npcs).empty?
 
-    escape_messages = ['You are already as far away as you can get', 'You retreat from combat', 'You sneak back out of combat']
-    until escape_messages.include?(DRC.bput('retreat', *(escape_messages + ['retreat', 'You try to back ', 'sneak', 'grip remains solid', 'You must stand first', 'grip on you', 'You stop advancing'])))
-      DRC.fix_standing
-    end
+    DRC.retreat
```

Comment:
should this be `retreat(ignored_npcs = [])`?

Reply:
it should be `DRC.retreat(ignored_npcs)`